src/lib.rs

```diff
@@ -101,6 +101,32 @@ macro_rules! tuple_meta_type {
     }
 }
 
+/// Construct a vector of `TypeParameter`s from pairs of the name and the concrete type.
```

Comment:
```suggestion
/// Construct a [`TypeParameter`] from the name and the concrete type.
```

Reply:
This macro actually does construct a vector, so it expands `named_type_params![(T, u16), (U, u32) ]` to `vec![ TypeParameter("T", MetaType::new::<u16>()), TypeParameter("U", MetaType::new::<u32>())]`